src/lib/pages/frontsite/PatronProfile/LoansList.js

```diff
@@ -0,0 +1,51 @@
+import { Pagination } from '@components/Pagination';
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { Container, Item } from 'semantic-ui-react';
+
+export default class LoanList extends Component {
+  render() {
+    const {
+      activePage,
+      isLoading,
+      onPageChange,
+      loans,
+      rowsPerPage,
+      renderListEntry,
+      noLoansCmp,
+    } = this.props;
+    console.log(loans.total, rowsPerPage);
```

Comment:
```suggestion
```

Reply:
Thanks, I had already remove in the next PR, but I am doing it now since there were other fixed to do.